c2pa_cbor/src/lib.rs

```diff
@@ -0,0 +1,1187 @@
+// Cargo.toml dependencies needed:
+// serde = { version = "1.0", features = ["derive"] }
+
+//! # C2PA CBOR Library
```

Comment:
I would move all the explanations in a readme in the src folder

Reply:
The stuff in the lib.rs shows up at the top level of the Rust Docs, so yes, we want the basic overview here.